src/main/java/org/jitsi/impl/protocol/xmpp/colibri/ColibriConferenceImpl.java

```diff
@@ -614,8 +614,8 @@ public void updateTransportInfo(
      * {@inheritDoc}
      */
     @Override
-    public void updateSourcesInfo(MediaSSRCMap ssrcs,
-                                  MediaSSRCGroupMap ssrcGroups,
+    public void updateSourcesInfo(MediaSourceMap ssrcs,
```

Comment:
I think we need to rename the code fully by also renaming variables from SSRCs to sources